src/smf.rs

```diff
@@ -830,6 +830,17 @@ impl<'a> EventIter<'a> {
         }
     }
 
+    /// Create an [`AbsoluteEventIter`] that starts at the current time.
```

Comment:
This contradicts the documentation of `AbsoluteEventIter`, which states that the ticks are relative to the start of the track.